config/locales/interface/input_elements/nl_supply_heat.yml

```diff
@@ -39,6 +39,13 @@ nl:
       description: Een warmtepomp produceert warmte uit elektriciteit en omgevingswarmte.
         Stel hier het vermogen aan collectieve warmtepompen in dat warmte levert aan
         warmtenetten.
+    capacity_of_energy_heat_boiler_electricity:
+      title: Elektrische boiler
+      short_description:
+      description: Dit schuifje beïnvloedt de capaciteit van elektrische boilers die warmte 
+        leveren aan warmtenetwerken. Een elektrische boiler produceert warmte uit elektriciteit. 
+        De ketel wordt vaak ingezet om pieken in het elektriciteitsnet op te vangen of 
+        als back-up ketel in een warmtenet.
```

Comment:
What do you mean with "pieken ... op te vangen"? Do you mean supply peaks on the electricity grid? This boiler is modelled as a dispatchable heat producer: it therefore responds to supply and demand on the **district heating network**, not on the electricity network.